### DISCUSSION TITLE
Inquiry about 'get_variables_along_trajectory' Attribute Error in OpenDrift

### DISCUSSION THREAD

**Discussion by HatYaz** [OpenDrift/opendrift#1219] [Q&A]
Dear OpenDrift Specialist,

I hope this message finds you well. I am currently working on a project involving the use of OpenDrift for trajectory simulations. While attempting to use the 'get_variables_along_trajectory' method as documented, I encountered an AttributeError.

Here is the relevant code snippet:
cur_traj = o.get_variables_along_trajectory(variables=['x_sea_water_velocity', 'y_sea_water_velocity'],
lons=lon_mtr,
lats=lat_mtr,
times=times_sim)

The error message I received is as follows:

AttributeError: 'OceanDrift' object has no attribute 'get_variables_along_trajectory'

I have reviewed the OpenDrift documentation, and it seems that 'get_variables_along_trajectory' should be a valid method for the 'OceanDrift' class. Could you please provide guidance on resolving this issue or clarify if there have been any changes in recent versions of OpenDrift that may affect the usage of this method?

Thank you in advance for your assistance.

**Comment by knutfrode** [comment]
Duplicate of https://github.com/OpenDrift/opendrift/issues/1220
